impl/src/util.rs

```diff
@@ -330,6 +331,10 @@ pub fn get_workspace_member_path(manifest_path: &Path, workspace_root: &Path) ->
   diff_paths(manifest_path.parent().unwrap(), workspace_root)
 }
 
+pub fn get_package_ident(package_name: &str, package_version: &str) -> String {
```

Comment:
Nit: Is just `package_ident` sufficient? `get` implies to me that we're looking it up from somewhere or something.

Also, probably good to define "package_ident" here as part of function documentation.